src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -7854,6 +7854,18 @@ void Assembler::vbroadcastsd(XMMRegister dst, Address src, int vector_len) {
   emit_operand(dst, src);
 }
 
+void Assembler::vbroadcastf128(XMMRegister dst, Address src, int vector_len) {
+  assert(VM_Version::supports_avx(), "");
+  assert(vector_len == AVX_256bit, "");
```

Comment:
Looks like "vector_len" can only be AVX_256bit. Do we really need a parameter then?

Reply:
your are right, for now it can only be AVX_256bit. But I think in the future other lengths will be used too. So we should have a more generic signature.